tests/manage/mcg/test_prefix_list.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.resources.objectbucket import OBC
+from ocs_ci.ocs.bucket_utils import (
+    write_random_objects_in_pod,
+    copy_objects,
+    list_objects_from_bucket,
+)
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.framework.pytest_customization.marks import (
+    bugzilla,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@bugzilla("2068110")
+class TestPrefixList:
+
+    """
+    Test S3 prefix list operations
+    """
+
+    @pytest.mark.parametrize(
+        argnames="first,second",
+        argvalues=[
+            pytest.param(
+                *["99", "990"],
+                marks=[],
+            ),
+            pytest.param(
+                *["11", "111"],
+                marks=[],
+            ),
+            pytest.param(
+                *["aa", "aaa"],
+                marks=[],
+            ),
+            pytest.param(
+                *["aa", "aa0"],
+                marks=[],
+            ),
+        ],
+    )
+    def test_prefix_list(
+        self,
+        awscli_pod_session,
+        bucket_factory,
+        test_directory_setup,
+        first,
+        second,
+    ):
+        """
+        Test s3 prefix uploads and list operations
+        """
+
+        bucketclass = {
+            "interface": "OC",
+            "backingstore_dict": {"aws": [(1, "eu-central-1")]},
+        }
+
+        bucket = bucket_factory(bucketclass=bucketclass)[0]
```

Comment:
why not just use default bucketclass?

Reply:
if I don't mention any bucketclass does it use default? @parikshithb

Reply:
Yes. it will use default.